### PR TITLE
Release 1.3.0-pre.0

### PR DESCRIPTION
**Pull request by kylef**
This pull request updates to Drafter 2.3.0-pre.0 and releases Protagonist as 1.3.0-pre.0.

### Changelog

This update now uses Drafter 2.3.0-pre.0. Please see [Drafter 2.3.0-pre.0](https://github.com/apiaryio/drafter/releases/tag/v2.3.0-pre.0) for the list of changes.

### CODE REVIEW

**Comment by honzajavorek** [comment]
What does exactly `pre-` release mean in the context of Drafter and Protagonist? Is it safe to depend on such version?

**Comment by pksunkara** [comment]
@honzajavorek Yes, it is. It is a pre-release because we want to patch some other bugs in this release.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "protagonist",
-  "version": "1.2.6",
+  "version": "1.3.0",
```
Comment (pksunkara) [inline package.json]: There is no `-pre.0` here.

**Comment by pksunkara** [comment]
Can't merge until the version number is correct.

**Comment by kylef** [comment]
@pksunkara resolved.

**Comment by pksunkara** [comment]
Please tag and release.

**Pull request merged**